x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/hooks/use_bulk_actions.ts

```diff
@@ -302,17 +304,22 @@ export function useBulkActions({
     featureIds,
     isAllSelected: bulkActionsState.isAllSelected,
   });
+
   const initialItems = useMemo(() => {
     return [...caseBulkActions, ...(featureIds?.includes('siem') ? [] : untrackBulkActions)];
   }, [caseBulkActions, featureIds, untrackBulkActions]);
   const bulkActions = useMemo(() => {
+    if (disableBulkActions) {
+      return [];
```

Comment:
nit: I am wondering about the naming of the prop. Based on this code we do not disable the bulk actions but we do not return any. Does it make sense to rename the prop or show the items and actually disable them?

Reply:
Agree, should I rename to `displayBulkActions`?

Reply:
Sounds good to me!